atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -81,7 +81,8 @@ def __init__(self, tasker, workflow, labels,
             raise RuntimeError("labels have to be dict")
         self.labels = labels
         self.dont_overwrite = dont_overwrite
-        self.aliases = aliases or {}
+        default_aliases = {k: v for k, v in _PREFERRED_LABELS}
```

Comment:
Personally I think it's fine for pre_add_labels_in_df to have its own copy as it will include labels that are not inspected by atomic-reactor.